src/edu/csus/ecs/pc2/core/execute/SandboxResults.java

```diff
@@ -33,8 +39,14 @@ public SandboxResults(File propertiesFilename) throws IOException {
         resultsProperties.load(fileInputStream);
     }
 
-    public String getProperty(String s) {
-        return resultsProperties.getProperty(s);
+    /**
+     * Return value for property from sandbox file.
+     * 
+     * @param key
+     * @return null or key value.
+     */
```

Comment:
This method doesn't return the "key value", it returns the "value stored under the specified key".  There's a difference.

Reply:
typo, fixed parameter doc (which is less work than class or method doc), which may be done later.